### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/tabs.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#650] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/tabs.html

让你的 Markdown 文件支持供选项卡。

https://theme-hope.vuejs.press/zh/guide/markdown/tabs.html

**Comment by fdmas** [comment]
如何在选项卡下面插入多个图表？我现在插入2个echarts图表后出现表1无法显示和表二宽度只有100px的问题。

**Reply by Mister-Hope** [reply]
不支持，echarts 的渲染要求正确获取它的容器尺寸，而选项卡里其他选项卡是默认隐藏的。

如果你自己开发一个网站你可以自己做一个监听器在切换tab的时候现渲染 echarts，但对于我们包装功能来说，没办法做这种“自动适配”

**Comment by qiuguolu** [comment]
很漂亮
